File: src/ProblemSolvingDayByDay/CodeForces/FridayRevise/Team.java
package ProblemSolvingDayByDay.CodeForces.FridayRevise;

import java.util.Scanner;

public class Team {
    public static void main(String[] args) {

        //todo: follow the  java Team..

        int a, b, c, count = 0;
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter your number: ");
        int n = sc.nextInt();

        for (int i = 0; i < n; i++) {
            a = sc.nextInt();
            b = sc.nextInt();
            c = sc.nextInt();
            if (a + b + c >= 2) {
                count++;
            }
        }
        System.out.println(count);
    }
}

File: src/ProblemSolvingDayByDay/CodeForces/Kotlin/Team.kt
package ProblemSolvingDayByDay.CodeForces.Kotlin

import java.util.*

fun main() {
    var testNum: Int
    var fFriend: Int
    var sFriend: Int
    var tFriend: Int
    var res: Int
    var count = 0;
    var sc = Scanner(System.`in`)
    testNum = sc.nextInt();

    for (i in 1..testNum) {
        fFriend = sc.nextInt()
        sFriend = sc.nextInt()
        tFriend = sc.nextInt()

        res = fFriend + sFriend + tFriend
        if (res >= 2) {
            count++
        }
    }
    println(count)
}